packages/frontend/src/constants.ts

```diff
@@ -33,50 +26,62 @@ export const DATA_CDN_URL = getServiceURL(
     __BUILDING_MODE__ === "production",
 );
 
-export interface AugmentedChain extends Chain {
-    logo: FunctionComponent<
-        SVGProps<SVGSVGElement> & { title?: string | undefined }
-    >;
-    iconBackgroundColor: string;
-    transport: Transport;
-}
-
-export const SUPPORTED_CHAINS =
+export const SUPPORTED_CHAINS: [Chain, ...Chain[]] =
     __BUILDING_MODE__ === "production"
-        ? ([
+        ? [
               {
                   ...gnosis,
-                  logo: GnosisLogo,
-                  iconBackgroundColor: "#04795b",
-                  transport: http(),
-              },
-          ] as const)
-        : ([
+                  rpcUrls: {
+                      default: {
+                          http: ["https://rpc.ankr.com/gnosis"],
```

Comment:
We can just leave "http()" here. Have a look at the default values for the RPCs. They're the same.